README.md

```diff
@@ -16,3 +16,32 @@ Command                                         | Description
 ```$ ./scripts/feeds/install device-manager```  | Install only the package device-manager
 
 
+### Building these packages using pre-compiled OpenWrt SDK for Ci40 (Marduk):
+
+1. First download the SDK from https://downloads.creatordev.io/pistachio/marduk/OpenWrt-SDK-0.9.4-pistachio-marduk_gcc-5.2.0_musl-1.1.11.Linux-x86_64.tar.bz.
+2. Extract the SDK and go to the extracted folder.
+3. Check the feeds.conf.default whether its having an entry to this https://github.com/CreatorKit/openwrt-ckt-feeds.git
```

Comment:
=> Check whether the feeds.conf.default file has an entry of https://github.com/CreatorKit/openwrt-ckt-feeds.git